### PR TITLE
feat: add community resources new pages

### PR DESCRIPTION
**Pull request by graz-dev**
Added new pages for Community resources

### CODE REVIEW

**Comment by graz-dev** [comment]
@fredmaggiowski here the changes to add the community resources page. Check it and let me know if it's ok for you. 
Before merge this PR we have to:
- Link the OSDay slides to the Talks md file (I need permission to push them in the community-resources repo)
- Decide if we need to add also media kit and presentations as pages in the community resources menu

**Comment by fredmaggiowski** [comment]
Can you share a screen of the resulting page in the PR so that we don't need to build it locally to see changes 😅 Thanks ❤️

File: pages/community.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+permalink: /community-resources/
+meta_description: Rönd Documentation is the developer guide to get started with the open source security authorization enforcement tool.
+---
+
+# Community Resources
+
+Welcome to the Community Resources page, your one-stop destination for all things about Rönd! 
+Here, you'll find an extensive collection of resources ranging from slides from past talks and workshops to a comprehensive media kit and various assets, all meticulously curated to help you further promote and engage with Rönd Technologies. Whether you're seeking to amplify your understanding of Rönd or looking to bolster your promotional efforts, you'll discover everything you need right at your fingertips. Dive in and explore the wealth of materials designed to empower your journey with Rönd Technologies.
+
+
```
Comment (fredmaggiowski) [inline pages/community.md:13]: We could write a paragraph here about media assets and presskit, what do you think? 🤔 

Also can we link in this page the Talks and Workshop pages?

**Comment by graz-dev** [comment]
Sure @fredmaggiowski here some screnshots:

image
image
image

It would be nice if you have some description also for the KCD munich talk and KCD UK workshop (maybe you have the abstract in your sessionize account?).

**Comment by graz-dev** [comment]
@fredmaggiowski I think that now we can close the PR. 
I added the link to the slides of the OSDay talk and the abstract for the KCD Munich one. ✌🏻

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**